### PR TITLE
Linting blacklisted json files.

### PR DESCRIPTION
**Pull request by zombiethoughts**
#### Summary

SUMMARY: none

#### Purpose of change

Linting blacklisted json files.

#### Describe the solution

Linted the following files...

data/json/mapgen/lab/lab_rooms.json
data/json/mapgen/lab/lab_rooms_wall.json
data/json/mapgen/lab/lab_trains.json

### CODE REVIEW

File: data/json/mapgen/lab/lab_rooms.json [hunk 1]
```diff
@@ -76,26 +84,19 @@
     "object": {
       "mapgensize": [ 7, 7 ],
       "rotation": [ 0, 3 ],
-      "rows": [
```
Comment (kevingranade) [inline data/json/mapgen/lab/lab_rooms.json]: Lets's skip linting files with small nested mapgen like this for now, it renders them unreadable.
I'll need to sort out a fix for this.

Comment (zombiethoughts) [reply data/json/mapgen/lab/lab_rooms.json]: No problem. I'll skip the mapgen ones until you give me an okay.

Comment (zombiethoughts) [reply data/json/mapgen/lab/lab_rooms.json]: Mind if I close the pull request then?

Comment (kevingranade) [reply data/json/mapgen/lab/lab_rooms.json]: Go ahead if it makes things easier.
And feel free to e.g. reorder entries in the blacklist if it makes it easier to keep track of your progress.

Comment (zombiethoughts) [reply data/json/mapgen/lab/lab_rooms.json]: Will do, thank you.

**Pull request closed without merging**